types/passport-totp/strategy.d.ts

```diff
@@ -0,0 +1,25 @@
+export = Strategy;
+
+type DoneCallback = (...args: any[]) => void;
+
+type SetupCallback = (user: any, done: DoneCallback) => void;
+
+interface StrategyOptions {
+    codeField?: string;
+    window?: number;
+}
+
+declare function Strategy(setup: SetupCallback): void;
+declare function Strategy(options: StrategyOptions, setup: SetupCallback): void;
+declare class Strategy {
+    constructor(setup: SetupCallback);
+    constructor(options: StrategyOptions, setup: SetupCallback);
+    _codeField: string;
+    _window: number;
+    _setup: SetupCallback;
```

Comment:
```suggestion
```
arent' those private? That's old code, so just assuming '_' convention here for private